docs/tutorial.rst

```diff
@@ -26,7 +26,7 @@ The Yellowbrick library is a diagnostic visualization platform for machine learn
 About the Data
 --------------
 
-This tutorial uses the mushrooms data from the Yellowbrick :doc:`api/datasets` module. Our objective is to predict if a mushroom is poisonous or edible based on its characteristics.
+This tutorial uses the mushrooms data from the Yellowbrick :doc:`api/datasets/index` module. Our objective is to predict if a mushroom is poisonous or edible based on its characteristics.
```

Comment:
This is the only substantive change in this document. The other changes are just from my linter removing extra whitespace.